### DISCUSSION TITLE
Configuration Mode when upright

### DISCUSSION THREAD

**Discussion by Andlix** [universam1/iSpindel#538] [Ideas]
Hi,

I think it would be an advantage if I could leave the circuit board in the housing.
A configuration mode would be useful for this, which starts when the iSpindel is on the lid and does not switch to sleep mode, so that the configuration website can be called up.
If you close this website or restart the iSpindel, it will run in normal mode again.

**Comment by pppedrillo** [comment]
Already discussed (do search through closed issues). Briefly: many often use their ispindels just as a thermometers stand on its cap, thus this feature already was rejected.
